torch/_dynamo/repro/after_aot.py

```diff
@@ -225,6 +226,35 @@ def inner_debug_fn(real_inputs):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
 
 
+def maybe_fbcode_instructions():
+    if is_fbcode:
+        return """\
+\"\"\"
+To run this script in fbcode:
+- Create a directory (//scripts/{your_unixname}/repro)
+- Put this file in scripts/{your_unixname}/repro/fx_graph_runnable.py
+- Add a TARGETS file that looks like the following
+- `buck2 run //scripts/{unixname}/repro:repro`
+
+NOTE: you may need additional deps to actually be able to run the script.
+```
+# Contents of TARGETS file
+load("@fbcode_macros//build_defs:python_binary.bzl", "python_binary")
+
+python_binary(
+    name = "repro",
+    main_src = "fx_graph_runnable.py",
+    deps = [
```

Comment:
could we at least have this include the deps for the operator libraries we print out at the top of the file ? 

https://github.com/pytorch/pytorch/blob/60542eeb33db9ed245fcc47b84d4e530bcdb4e44/torch/_dynamo/debug_utils.py#L48-L55

Right now the file will never run bc we always include those imports.

Reply:
sounds good!